### DISCUSSION TITLE
[PSD6] Exercise 2.13

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#147] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210016885-90639017-7fcf-4dcb-8c15-277bf993889d.png)

**Comment by Elia-Belli** [comment]
- (a) $AC + \bar A \bar B C$
       $C(A+ \bar A \bar B) = C((A+\bar A)(A+ \bar B)) = C(A+\bar B) = AC + C\bar B$

- (b) $\bar A \bar B+ \bar A B \bar C + \bar A C$
       $\bar A (\bar B + B \bar C + C) = \bar A ((B+\bar B)(\bar B + \bar C)+C) = \bar A (\bar B + \bar C+C) = \bar A$

- (c) $\bar A \bar B \bar C \bar D + A\bar B \bar C + A \bar B C \bar D + ABD + \bar A \bar B C \bar D + B \bar C D + \bar A$
       $\bar A + (ABD + AB\bar C D + \bar A B \bar C D) + A\bar B \bar C + A \bar B C \bar D=$
       $\bar A + ABD + A\bar B (\bar C + C \bar D) =$
       $\bar A + ABD + A\bar B ((C + \bar C)(\bar C + \bar D)) = \bar A + ABD + A\bar B \bar C + A \bar B \bar D =$
       $\bar A + BD + \bar B \bar C + \bar B \bar D$

**Comment by Emanuele-Dambrosio** [comment]
[A](https://www.boolean-algebra.com/?q=QUMre0FCfUM=)
[B](https://www.boolean-algebra.com/?q=e0FCfSt7QX1Ce0N9K3tBfUM=)
[C](https://www.boolean-algebra.com/?q=e0FCQ0R9K0F7QkN9K0F7Qn1De0R9K0FCQyt7QUJ9Q3tEfStCe0N9RCtB)

**Comment by Elia-Belli** [comment]
@Emanuele-Dambrosio hai copiato male il testo in tutte e 3, scrivere $\overline {AB} \not= \bar A \bar B$

**Comment by Emanuele-Dambrosio** [comment]
è la stessa cosa, lo puoi verificare con velocemente facendo le tabelle della verità.

**Comment by Elia-Belli** [comment]
@Emanuele-Dambrosio puoi osservare che le tabelle differiscono, se trovi $\overline {AB}$ devi applicare De Morgan e diventa $\bar A + \bar B$
AB $\overline {AB}$ $\bar A \bar B$
00 1 1
01 1 0
10 1 0
11 0 0

**Comment by Emanuele-Dambrosio** [comment]
Hai ragione. Grazie!

**Comment by FeddyLix17** [comment]
tutto corretto

**Comment by fraxb04** [comment]
errato, dato che !C+C =  1 e !B + 1 = 1....
quindi verrebbe !A...
ps: Exyss ha sbagliato a scrivere nei suoi appunti una parte dei teoremi (ha detto prima o poi la correggerà)

**Comment by FeddyLix17** [comment]
@Elia-Belli mi sono accorto pure io adesso dell'errore notificato da @fraxb04 
la semplificazione dell'espressione booleana **(b)** dovrebbe essere quindi:
A&#x0304;B&#x0304; + A&#x0304;BC&#x0304; + A&#x0304;C
A&#x0304;( B&#x0304;+BC&#x0304;+C )
A&#x0304;( B&#x0304;+C&#x0304;+ C )
A&#x0304;( B&#x0304;+1 )
A&#x0304;( 1 )
A&#x0304;

**Comment by Elia-Belli** [comment]
Avete ragione, errore di distrazione
